cmd/dashboard/main.go

```diff
@@ -47,6 +47,7 @@ var (
 	logFormat          = flag.String("log-format", "json", "Format for log output (json or console)")
 	streamLogs         = flag.Bool("stream-logs", false, "Enable log streaming instead of polling")
 	externalLogs       = flag.String("external-logs", "", "External logs provider url")
+	xFrameOptions      = flag.String("xframe-options", "DENY", "Whether or not a browser should be allowed to render a page in a <frame>, <iframe>, <embed> or <object>")
```

Comment:
`flag.String("x-frame-options"…` so it's consistent with the header name and less likely to cause confusion / error

we should update the description too, maybe something like `Value to set for the X-Frame-Options response header ("" to omit it)`

Reply:
Just updated in a new commit.